app/src/main/java/org/jellyfin/mobile/fragment/WebViewFragment.kt

```diff
@@ -185,8 +186,15 @@ class WebViewFragment : Fragment() {
     }
 
     fun onSelectServer(error: Boolean = false) {
-        if (error) activity?.replaceFragment<ConnectFragment>(Bundle().apply { putBoolean(Constants.FRAGMENT_CONNECT_EXTRA_ERROR, true) })
-        else activity?.addFragment<ConnectFragment>()
+        activity?.run {
+            if (lifecycle.currentState.isAtLeast(Lifecycle.State.RESUMED)) {
+                if (error) {
+                    replaceFragment<ConnectFragment>(Bundle().apply { putBoolean(Constants.FRAGMENT_CONNECT_EXTRA_ERROR, true) })
```

Comment:
```suggestion
                    replaceFragment<ConnectFragment>(Bundle().apply {
                        putBoolean(Constants.FRAGMENT_CONNECT_EXTRA_ERROR, true)
                    })
```

Personal preference, I hate long lines 😂

Reply:
I already thought about that, but found the apply on the inside parameter a bit weird. Maybe I just add an extra variable for the bundle above?

Reply:
Yeah that should work, additionally we can use the if-statement to add to the bundle instead of the 2 identical addFragment calls

Reply:
It's not the same statement though :P
Anyway, fixed, gonna merge then.

Reply:
Yeah just noticed, whoops